pom.xml

```diff
@@ -76,7 +76,7 @@
     </scm>
     <properties>
         <awsjavasdk.version>${project.version}</awsjavasdk.version>
-        <jackson.version>2.9.3</jackson.version>
+        <jackson.version>2.9.6</jackson.version>
```

Comment:
should we use 2.9.5 though, from their github page, 2.9.5 is the latest stable version (not sure if it's out of date)

https://github.com/FasterXML/jackson

Reply:
I would guess he just hasn't updated the README on that repo. 2.9.6 is in Maven so I can't imagine it not being stable.